src/cluster.h

```diff
@@ -37,6 +40,14 @@ struct clusterState;
 #define CLUSTER_MODULE_FLAG_NO_FAILOVER (1 << 1)
 #define CLUSTER_MODULE_FLAG_NO_REDIRECTION (1 << 2)
 
+/* For clusterBroadcastPong */
+#define CLUSTER_BROADCAST_ALL 0            /* All known instances. */
+#define CLUSTER_BROADCAST_LOCAL_REPLICAS 1 /* All replicas in my primary-replicas ring. */
+
+/* Slot migration log configurations */
+#define CLUSTER_SLOT_MIGRATION_LOG_MAX 1000
+#define CLUSTER_SLOT_MIGRATION_LOG_TTL_MILLIS 3600000 /* 1 hour */
```

Comment:
You can remove these 2 macro definitions, they are mentioned in valkey.conf and default values are set in config.c.